src/code42cli/cmds/cases.py

```diff
@@ -20,20 +19,23 @@
 case_number_option = click.option(
     "--case-number", type=int, help="The number assigned to the case.", required=True
 )
-name_option = click.option("--name", help="Name of the case.",)
-assignee_option = click.option("--assignee", help="User UID of the assignee.")
-description_option = click.option("--description", help="Description of the case.")
-findings_option = click.option("--findings", help="Findings on the case.")
-subject_option = click.option("--subject", help="User UID of a subject of the case.")
+name_option = click.option("--name", help="The name of the case.",)
+assignee_option = click.option(
+    "--assignee", help="The UID of the user to assign to the case."
```

Comment:
`to assign to the case` makes sense for both `update` and `create`, where this is used.